src/wp-includes/class-wp-theme-json.php

```diff
@@ -1717,7 +1717,12 @@ protected static function compute_style_properties( $styles, $settings = array()
 	 * @return string|array Style property value.
 	 */
 	protected static function get_property_value( $styles, $path, $theme_json = null ) {
-		$value = _wp_array_get( $styles, $path );
+		$value = _wp_array_get( $styles, $path, '' );
+
+		// It doesn't make sense to process the value further.
+		if ( ( '' === $value ) || ( null === $value ) ) {
```

Comment:
Nitpick: the extra parentheses are unncessary here.

```suggestion
		if ( '' === $value || null === $value ) {
```